src/parser.cc

```diff
@@ -932,6 +933,15 @@ bool readXml(TiXmlElement *_xml, ElementPtr _sdf, Errors &_errors)
 
           // Get the config.xml filename
           filename = getModelFilePath(modelPath);
+
+          if (filename.empty())
+          {
+            _errors.push_back({ErrorCode::URI_LOOKUP,
```

Comment:
This error code seems like a bit of a stretch in this situation, because there isn't really a URI that the user provided and couldn't be found.  But the message is descriptive enough.

Reply:
do you think `URI_INVALID` is better? I wasn't sure what to put

Reply:
Both the `URI` errors sound to me like there's a mistake within an SDF file. But I don't feel too strongly about this. Do you think adding a new error code would be too much? Something like `MISSING_CONFIG`?

Reply:
`URI_LOOKUP` is used earlier in this same block of code if a model matching the specified `uri` can't be found, so I think it is consistent to use it here as well

* https://github.com/osrf/sdformat/blob/sdformat9_9.3.0/src/parser.cc#L912

Reply:
Ok, it makes sense to me then. The URI specifies the path that must contain both `.sdf` and `.config` files.